exporter/datadogexporter/blocklister.go

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datadogexporter
+
+import (
+	"regexp"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/exportable/pb"
+)
+
+// From: https://github.com/DataDog/datadog-agent/blob/a6872e436681ea2136cf8a67465e99fdb4450519/pkg/trace/filters/blacklister.go#L15-L19
+// Blocklister holds a list of regular expressions which will match resources
+// on spans that should be dropped.
+type Blocklister struct {
```

Comment:
Denylist is a term that is used in this context more frequently. Maybe that's better?

Reply:
makes sense to me, updated